### PR TITLE
Audit descriptions

### PR DESCRIPTION
**Pull request by sj26**
Some attempts at more event types in descriptions:

screen shot 2017-07-20 at 5 41 35 pm

I was playing around with some styles, too, and omitting fields without data. React won't let me return a dt/dd without a parent so I had to make multiple dls :-(((

Also "an user", hmm. Might need a special case.

### CODE REVIEW

File: app/components/audit_log/row.js [hunk 1]
```diff
@@ -204,6 +246,17 @@ export default Relay.createContainer(AuditLogRow, {
...
+            ...on Team {
+              name
+            }
+            ...on TeamMember {
+              team { name }
+              user { name }
+            }
+            ...on TeamPipeline {
+              team { name }
+              pipeline { name }
```
Comment (ticky) [inline app/components/audit_log/row.js]: I know it doesn’t affect much but for the sake of future diffs can we keep each query property on a separate line? 😄

Comment (sj26) [inline app/components/audit_log/row.js]: Considering I'm a stickler for hanging commas for the same reason, I endorse this product and/or service. :+1:

File: app/components/audit_log/Drawer/subjectSection.js [hunk 2]
```diff
@@ -26,28 +26,56 @@ class AuditLogSubjectSection extends React.PureComponent {
...
-            {subject.type}
-          </dd>
-          <dt className="mt1 dark-gray">
             Subject Name
           </dt>
           <dd className="ml0">
-            {subject.name}
+            {name}
           </dd>
+        </dl>
```
Comment (sj26) [inline app/components/audit_log/Drawer/subjectSection.js]: Do we need key props if the collection will never change? That's just to optimise when rows are likely to change around other rows, right?

Comment (sj26) [inline app/components/audit_log/Drawer/subjectSection.js]: And of course I can use arrays 🤦🏻‍♂️

**Comment by sj26** [comment]
Looks like this, now:

screen shot 2017-07-21 at 4 24 51 pm

**Comment by ticky** [comment]
Heck yeah ✌🏼

**Pull request merged**